specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/examples/ApiManagementCreateApiVersionSet.json

```diff
@@ -0,0 +1,31 @@
+{
+    "parameters": {
+      "serviceName": "apimService1",
+      "resourceGroupName": "rg1",
+      "api-version": "2017-03-01",
+      "subscriptionId": "subid",
+      "apiVersionSetId": "api1",
```

Comment:
>apiVersionSetId [](start = 7, length = 15)

this value must match with the value in the path `{apiVersionSetId}` or `{versionSetId}`